lib/plugins/tag/code.js

```diff
@@ -118,9 +118,11 @@ function parseArgs(args) {
 
 module.exports = ctx => function codeTag(args, content) {
 
+  const options = parseArgs(args);
+
   // If neither highlight.js nor prism.js is enabled, return escaped code directly
   if (!ctx.extend.highlight.query(ctx.config.syntax_highlighter)) {
-    return `<pre><code>${escapeHTML(content)}</code></pre>`;
+    return `<pre><code${options.lang !== undefined ? ` class="language-${options.lang}"` : ''}>${escapeHTML(content)}</code></pre>`;
```

Comment:
**Identical blocks of code found** on this line (125 and 140). I think you can create this line as separate variable and use it both places.